### DISCUSSION TITLE
Write multiple output files with one input file

### DISCUSSION THREAD

**Discussion by jheyduk** [hairyhenderson/gomplate#1803] [Q&A] [answered]
Hello all,
I would do something like this:
inputfile: config.json
`[
   {
      "name":"appset-1",
      "config":"sample value"
   },
   {
      "name":"appset-2",
      "config":"sample value"
   }
]`

My goal is to generate two output files: appset-1.yaml, appset-2.yaml

Is there anyway to achieve this? I don't want to create one input file for each output file.

**Comment by jheyduk** [comment, marked as answer]
I found the solution in [documentation](https://docs.gomplate.ca/usage/#--output-map)
Thank you very much for this great tool. This tool is a kind of Jack of all trades ;-)

**Reply by hairyhenderson** [reply]
Glad to hear you found a solution! And glad to hear you're finding gomplate useful 😁
